### PR TITLE
Add jamajama1.md

### PR DESCRIPTION
**Pull request by jamajama1**
Fixes #

### RawGit preview link

https://github.com/jamajama1/jamajama1.github.io/blob/add-jamajama1-git-profile/pages/vi/profiles/jamajama1.md

### CODE REVIEW

**Comment by singharpita** [comment]
@jamajama1 You are supposed to put link of raw.githack
Link for you PR will be https://raw.githack.com/jamajama1/jamajama1.github.io/add-jamajama1-git-profile/#!pages/vi/profiles/jamajama1.md

**Comment by jamajama1** [comment]
Hello Singharpita,

Sorry about the link. I tried finding it but wouldn't work for me. Should I
update the pull request or create a new one?

Best wishes,
Jama

File: pages/vi/profiles/jamajama1.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+![alt text](http://arts-sciences.buffalo.edu/content/arts-sciences/linguistics/about/linguistics-at-ub/jcr:content/par/image.img.688.auto.q80.jpg/1506973707788.jpg "University at Buffalo")
+
+# About Me
+
+I am a software engineer attending University at Buffalo at the School of Engineering and Applied Science. 
+I am working on improving my skills as an engineer and currently pursuing internship opportunities. 
+I consider myself a web scraping expert. I hope to work with you all in the near future. You can find my 
+LinkedIn page [here](https://www.linkedin.com/in/jama-jama-007915162). Feel free to connect with me.
+
+Languages I have worked with:
```
Comment (svlesiv) [inline pages/vi/profiles/jamajama1.md]: Hi @jamajama1. Everything looks good, except the list:

Screen Shot 2019-04-16 at 5 41 18 PM
To fix that you could define `Languages I have worked with:` with a heading markup ( `## Languages I have worked with: `) or add an empty line after that phrase.

**Comment by ChiragAwale** [comment]
Inactive.

**Pull request closed without merging**